src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,50 @@
+import cn from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+import { Sex, SearchField, SortParam } from '../types';
+
+type Centuries = '16' | '17' | '18' | '19' | '20';
+
+const centuriesList: Centuries[] = ['16', '17', '18', '19', '20'];
```

Comment:
If you need an array with exactly known values you can use the tuple 
https://www.w3schools.com/typescript/typescript_tuples.php